src/components/dialog_preference/index.tsx

```diff
@@ -15,6 +18,7 @@ import { useRecoilValue, useResetRecoilState } from 'recoil'
 
 const PreferenceDialog: FC = () => {
   const isPreferenceDialogOpened = useRecoilValue(isPreferenceDialogOpenedState)
+  const Preferences = useRecoilValue(PreferenceState)
```

Comment:
```suggestion
  const preferences = useRecoilValue(preferenceState)
```
Like this!

Can you also change it from
`preferences` -> `preference`
```suggestion
  const preference = useRecoilValue(preferenceState)
```